mm.js

```diff
@@ -0,0 +1,8 @@
+let number = prompt(" Enter the Grade")
+if (number >= 90) { console.log("the Grade is A"); }
+else if (number >= 80 || number == 89) {
```

Comment:
you dont really need the or because the 90 above makes it so you only need the number>=90